lib/configuration.js

```diff
@@ -229,6 +223,21 @@ Configuration.prototype._checkLocalServiceContext = function() {
  * @returns {Undefined} - does not return anything
  */
 Configuration.prototype._gatherLocalConfiguration = function() {
+  if (this._givenConfiguration.force === true) {
+    this._shouldReportErrorsToAPI = true;
+  } else if (has(this._givenConfiguration, 'force')) {
+    throw new Error('config.force must be a boolean');
+  } else {
+    this._shouldReportErrorsToAPI = env.NODE_ENV === 'production';
+  }
+  if (!this._shouldReportErrorsToAPI) {
+    this._logger.warn([
+      'Stackdriver error reporting client has not been configured to send',
+      'errors, please check the NODE_ENV environment variable and make sure it',
+      'is set to production or the force property is set to true in the',
```

Comment:
@steren lol thanks for catch again -- still getting my first coffee on over here